api/settings/settings.go

```diff
@@ -1,11 +1,9 @@
 package settings
```

Comment:
This file lets us retrieve settings from multiple places. Env is only used for testing locally; Vault and KMS are both options for Production